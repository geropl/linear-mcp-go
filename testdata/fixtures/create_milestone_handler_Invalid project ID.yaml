---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 440
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tmutation ProjectMilestoneCreate($input: ProjectMilestoneCreateInput!) {\n\t\t\tprojectMilestoneCreate(input: $input) {\n\t\t\t\tsuccess\n\t\t\t\tprojectMilestone {\n\t\t\t\t\tid\n\t\t\t\t\tname\n\t\t\t\t\tdescription\n\t\t\t\t\ttargetDate\n\t\t\t\t\tproject {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tname\n\t\t\t\t\t}\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"input":{"name":"Test Milestone 3.1","projectId":"invalid-project-id"}}}'
        form: {}
        headers:
            Content-Type:
                - application/json
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"errors":[{"message":"Argument Validation Error","path":["projectMilestoneCreate"],"locations":[{"line":3,"column":4}],"extensions":{"code":"INVALID_INPUT","validationErrors":[{"target":{"name":"Test Milestone 3.1","projectId":"invalid-project-id"},"value":"invalid-project-id","property":"projectId","children":[],"constraints":{"isUuid":"projectId must be a UUID"}}],"type":"invalid input","userError":true,"userPresentableMessage":"projectId must be a UUID."}}],"data":null}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"1df-TEVUGn11CxiIlKsKcBOVc2L/nlo"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 0s
