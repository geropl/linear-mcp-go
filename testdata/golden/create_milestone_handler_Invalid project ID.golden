err: 'Failed to create milestone: GraphQL error: Argument Validation Error'
output: ""
